tensorbay/opendataset/__init__.py

```diff
@@ -76,6 +77,7 @@
     "NeolixOD",
     "Newsgroups20",
     "NightOwls",
+    "nuScenes",
```

Comment:
`nuScenes` start with a lower case `n`, it should be put before the upper case letters.